chapter_linear-networks/linear-regression-scratch_vn.md

```diff
@@ -116,22 +116,25 @@ d2l.plt.scatter(features[:, 1].asnumpy(), labels.asnumpy(), 1);
 ## Reading the Dataset
 -->
 
-## *dịch tiêu đề phía trên*
+## Thực hiện đọc tập dữ liệu
 
 <!--
 Recall that training models consists of making multiple passes over the dataset, grabbing one minibatch of examples at a time, and using them to update our model.
 Since this process is so fundamental to training machine learning algorithms, its worth defining a utility function to shuffle the data and access it in minibatches.
 -->
 
-*dịch đoạn phía trên*
+Nhắc lại là các mô hình huấn luyện bao gồm việc tách tập tập dữ liệu thành nhiều phần (các mininbatch), việc lần lượt đọc từng phần của tập dữ liệu mẫu, và việc sử dụng chúng để cập nhật mô hình của chúng ta. 
```

Comment:
```suggestion
Xin nhắc lại rằng việc huấn luyện mô hình bao gồm nhiều bước, mỗi bước sử dụng một phần của tập dữ liệu mẫu (mỗi phần gọi là một mininbatch hay một lô) để cập nhật tham số của mô hình. 
```

Mininbatching là phân lô :)

Reply:
@tanduong mình biết có một số tài liệu dịch `batch` là `lô`. Trong cuốn này từ đầu nhóm đã không dịch mà giữ nguyên là `batch`.

Chúng ta có thể bàn thêm việc có dịch `batch` thành `lô` không nhưng có lẽ ở một thread khác. Ở đây, để đảm bảo tính  nhất quán cho  bản dịch, chúng ta giữ nguyên là `batch`. Mọi thay đổi cần làm ở mức toàn cục, không phải chỉ ở phần này.

Nếu nhóm thống nhất dịch `batch` là `lô` thì sẽ thay đổi toàn bộ cả các phần khác nữa.

cc @lkhphuc